articles/key-vault/key-vault-ovw-storage-keys.md

```diff
@@ -102,7 +102,7 @@ Key Vault needs permissions to *list* and *regenerate* keys for a storage accoun
     
      or
      
-    `Get-AzureRmADServicePrincipal -SearchString "AzureKeyVault"`
```

Comment:
Hi Adam, I think that we should remove this line completely to be honest, I think it just creates confusion as to how to get the Id. Also, people can add names and so there could be multiple "Azure Key Vault" just like there is multiple "AzureKeyVault". Let me know what you think.

Reply:
@AdamCLarsen, @fhokholdMSFT - thanks for talking this through. This is another important way our content gets improved.
~ Bruce

Reply:
@fhokholdMSFT I agree, especially if I was to be scripting this.  Then I really should only use the GUID.

It wasn't clear to me just reading it, that the GUID was an Azure global GUID.  And not until I got the sample completed and saw it in the portal, did I realized what that step was actually doing.